src/metabase/models/revision.clj

```diff
@@ -36,7 +36,9 @@
 
 (defmethod revert-to-revision! :default
   [model id _user-id serialized-instance]
-  (t2/update! model id serialized-instance))
+  (let [valid-columns   (keys (t2/select-one (t2/table-name model) :id id))
+        revert-instance (select-keys serialized-instance valid-columns)]
+    (t2/update! model id revert-instance)))
```

Comment:
Looks good. Do you think a comment here about why we're doing that, and what other errors can happen makes sense here?